File: src/main/kotlin/wowsft/model/gameparams/ship/component/airarmament/AirArmament.java
package wowsft.model.gameparams.ship.component.airarmament;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class AirArmament
{
    private float auraCoeff;
    private float deckPlaceCount;
    private boolean isIndependentLaunchpad;
    private float launchPrepareTime;
    private String launchpadType;
    private float planesReserveCapacity;
}

File: src/main/kotlin/wowsft/model/gameparams/ship/component/airarmament/AirArmament.kt
package wowsft.model.gameparams.ship.component.airarmament

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class AirArmament
{
    var auraCoeff = 0.0
    var deckPlaceCount = 0.0
    var isIndependentLaunchpad = false
    var launchPrepareTime = 0.0
    var launchpadType: String? = null
    var planesReserveCapacity = 0.0
}
